File: 程式語言 V2答案檔/C#/207.cs
using System;
class Class1 {
    public static int isPrime (int input) {
        for (int i=2; i<=Math.Sqrt(input); i++) {
            if (input % i == 0)  return 0;
        }
        return 1;
    }
    static void Main() {
      	int input = Convert.ToInt32(Console.ReadLine());
        if (isPrime(input) == 1) {
            Console.WriteLine($"{input} is a prime number");
        }
        else {
            Console.WriteLine($"{input} is not a prime number");
        }
    }
}

File: 程式語言 V2答案檔/C++/207.cpp
#include <iostream>
using namespace std;

int main()
{
    int num, i;
    bool isPrime = true;
    cin >> num;
    for (i = 2; i * i <= num; i++) {
        if (i == num) {
            break;
        }
        else if (num % i == 0) {
            isPrime = false;
            break;
        }
    }
    if (isPrime)
        cout << num << " is a prime number";
    else
        cout << num << " is not a prime number";
    return (0);
}